modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/dto/v1_0/CartItemDTOConverter.java

```diff
@@ -95,6 +98,24 @@ public CartItem toDTO(DTOConverterContext dtoConverterContext)
 		};
 	}
 
+	private String[] _getErrorMessage(
```

Comment:
@luca-pellizzon  I think we should call this method the plural form since its returning an string array, even though its only one item in the array.
`_getErrorMessages`

Reply:
Done